crawler.py

```diff
@@ -87,15 +89,26 @@ def test_crash(driver):
 var crash = badptr.contents;""")
 
 
-def get_domain_list(n_sites):
+def get_domain_list(n_sites, exclude_option):
     """Get the top n sites from the tranco list"""
-    tr = Tranco(cache=False)
-
-    if not n_sites:
```

Comment:
Looks like a merge went wrong ... This shouldn't have gotten removed.

Reply:
I see that f4b887771a3f73c1b4cbdfb05af51a697166e4b6 is not a merge commit but contains some of my changes from master.